library/blivet.py

```diff
@@ -526,8 +524,8 @@ def _create(self):
                                                                    self._volume.get("raid_device_count"),
                                                                    self._volume.get("raid_spare_count"))
 
-        chunk_size = Size(self._volume.get("raid_chunk_size", MD_CHUNK_SIZE))
-        metadata_version = self._volume.get("raid_metadata_version", "default")
+        chunk_size = Size(self._volume.get("raid_chunk_size"))
```

Comment:
I don't think this will go well. Shouldn't it be `Size(...)` only if it has a size-like value, otherwise `None` to let blivet pick the default?

Reply:
True.
Fixed: None value will now prevent casting to Size. Added handling None value later in code.